test/time_tracker/test_helpers.clj

```diff
@@ -6,7 +6,14 @@
             [gniazdo.core :as ws]
             [clojure.test :as test]
             [clojure.spec.test :as stest]
-            [time-tracker.util :as util]))
+            [time-tracker.util :as util]
+            [clojure.string :as str]))
+
+(def test-port (Integer/parseInt (util/from-config :port)))
```

Comment:
My skimming didn't notice that this `def` had a runtime function in it.

I'd upgrade @jysandy 's comment to "Never do *anything* at compile time... unless absolutely necessary." It will always be a cause for confusion ...and probably brokenness.